### PR TITLE
run API flask application using uWSGI.

### PR DESCRIPTION
**Pull request by UiP9AV6Y**
instead of running flask with the builtin development server, use
a proper application server. uWSGI can be configured either via
files or environment variables; the chart has been extended to
allow environment variables to be injected into the container.

see #34

### CODE REVIEW

**Comment by Jaydee94** [comment]
Hey @UiP9AV6Y , thanks for your approach to use uWSGI.
We already opened an issue to use the FastAPI Framework instead of Flask. We will discuss if we use FastAPI or uWSGI and decide what to to with your PR.
Thanks a lot for your commitment.

**Comment by UiP9AV6Y** [comment]
yes, i saw that issue but didn't think much of it as fastAPI is a framework operating on the same stack level ask flask. but apperently the integration into an application server is done via ASGI, which uWSGI does not support (from a quick search there seems to be some third party effort).

i thinks the base concept of this PR can still be reused when switching to fastAPI and uvicorn/daphne/hypercorn.

let me know about the outcome, as it also affects my other PRs.

File: api/run.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+from os import environ
+
+from app import create_app
+
+app = create_app()
+bind_address = "127.0.0.1"
+listen_port = "5000"
+
+if "PORT" in environ:
```
Comment (FSchumacher) [inline api/run.py:11]: What do you think of
```python
bind_address = envrion.get("HOST", "127.0.0.1")
listen_port = int(environ.get("PORT", "5000"))
```

File: api/run.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    listen_port = int(environ["PORT"])
+
+if "HOST" in environ:
+    bind_address = environ["HOST"]
+
+if __name__ == "__main__":
+    app.run(debug=True, host=bind_address, port=listen_port)
```
Comment (FSchumacher) [inline api/run.py:18]: The debug mode could be made configurable, don't you think?

Comment (UiP9AV6Y) [reply api/run.py:18]: depends. this code is not run in the
context of uWSGI, but when you run `run.py` from a shell, which is usually when you run it locally for testing.

**Review by Jaydee94 — APPROVED** [review]

**Pull request merged**